src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -706,7 +706,7 @@
   "block.structurize.side_framed_jungle_brick_timber_frame": "Side Framed Jungle Brick Timber Frame",
   "block.structurize.horizontal_plain_spruce_stone_timber_frame": "Plain Horizontal Spruce Stone Timber Frame",
   "block.structurize.one_crossed_lr_dark_oak_stone_timber_frame": "Left Right Crossed Dark Oak Stone Timber Frame",
-  "structurize.command.ls.destroy.done": "Destroying session of player: %s",
+  "structurize.command.ls.destroy.done": "Destroying session of player %s.",
```

Comment:
Same above

Reply:
What do you mean? The above comment was about not adding colons in the config file. This looks like it's in response to a command and removes a colon.

Reply:
Sorry, was referring to created session comment